README.md

```diff
@@ -66,6 +66,24 @@ Option | Description | Example values
 `--interval` | The time between status updates (in seconds) | `1800` `86400`
 `--tweets-csv` | File path or URL for tweets.csv (required) | `./tweets.csv` `https://example.com/path/to/tweets.csv`
 
+# Using Docker
+### 1. Build
+```
+$ docker build -t markov-bot .
+```
+
+### 2. Run
+If you put `tweets.csv` in project root directory, for example, you can run with the following command.
+```shell
+$ cd markov-bot
+$ docker run -d -e "MARKOV_BOT_CONSUMER_KEY=<YOUR_CONSUMER_KEY>" \
+-e "MARKOV_BOT_CONSUMER_SECRET=<YOUR_CONSUMER_SECRET>" \
+-e "MARKOV_BOT_ACCESS_TOKEN=<YOUR_ACCESS_TOKEN>" \
+-e "MARKOV_BOT_ACCESS_TOKEN_SECRET=<YOUR_TOKEN_SECRET>" \
+-v $(pwd)/tweets.csv:/markov-bot/tweets.csv \
+markov-bot -- --tweets-csv ./tweets.csv --interval 1800
```

Comment:
It does not work without `--` (I spent a lot of time finding this...)

I tried using `--` in `ENTRYPOINT` but it does not work too.
```dockerfile
# not work
ENTRYPOINT ["stack", "exec", "markov-bot-exe", "--"]
docker run ... markov-bot --tweets-csv ./tweets.csv --interval 1800
```

Reply:
`["stack", "exec", "--", "markov-bot-exe"]` will do.